pkg/graph/schema.graphql

```diff
@@ -1606,10 +1669,11 @@ type Mutation {
   sendFeedbackEmail(input: SendFeedbackEmailInput!): String
   sendCantFindSomethingEmail(input: SendCantFindSomethingEmailInput!): String
   sendReportAProblemEmail(input: SendReportAProblemEmailInput!): String
-  
   createTRBRequest(requestType: TRBRequestType!): TRBRequest!
-  
   updateTRBRequest(id: UUID!, changes: TRBRequestChanges): TRBRequest!
+  createTRBRequestAttendee(input: CreateTRBRequestAttendeeInput!): TRBRequestAttendeePayload
+  updateTRBRequestAttendee(input: UpdateTRBRequestAttendeeInput!): TRBRequestAttendeePayload
```

Comment:
If the fields for create and update are pretty similar,  you could simplify the amount of input types here by adding another parameter, something like 
`updateTRBRequestAttendee((id: UUID!, input: UpdateTRBRequestAttendeeInput)`

I do wonder though if fields like euaUserId and trbrequestID should be left out of the update type. I don't think it would make sense to change a users EUA here, or the request id. Those operations seem more suited to creating a new record, or deleting incorrect records.

Reply:
> If the fields for create and update are pretty similar, you could simplify the amount of input types here by adding another parameter, something like updateTRBRequestAttendee((id: UUID!, input: UpdateTRBRequestAttendeeInput)

I feel like just having one input object that contains everything is cleaner UX from the standpoint of a person consuming the API

> I do wonder though if fields like euaUserId and trbrequestID should be left out of the update type.

Ah yeah, good call. I got a little copy/paste happy on that one.